etc/pr_check/main.go

```diff
@@ -0,0 +1,149 @@
+package main
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"time"
+
+	"github.com/google/go-github/v54/github"
+	flag "github.com/spf13/pflag"
+	"golang.org/x/oauth2"
+)
+
+var interestingPRs []int = []int{
+	4560, // Support pachctl mount on GKE by wsxiaoys
+	8170, // Delete dead code in pach, by seslattery
+	6325, // Fix joins example, by me
+	7374, // Add tool providing pachctl cli on debug dumps, by PFedak (do we want this?)
+	8068, // Add Godoc documentation, by PFedak (pretty sure we want this)
+}
+
+var prsToClose []int = []int{
+	3607, // Storage layer design, by brycemcanally
+}
+
+var (
+	startDay  string
+	endDay    string
+	authors   []string
+	prsToScan []int
+
+	missingJira bool
+	lastMinute  bool
+	printPRs    bool
+	status      bool
+	comment     bool
+	help        bool
+)
+
+func init() {
+	flag.CommandLine.SortFlags = false
+	flag.StringVar(&startDay, "start-day", "", "Only scan PRs created after the start of the given day (must be YYYY-MM-DD).")
```

Comment:
What time in what time zone does a "day" start?  I think this should just be a time.Time.  An additional nice cleanup would be finding where pachctl accepts times, writing a src/internal/ppflag module (pachyderm pflag lol), and adding a ppflag.TimeVar method ;)